src/shared/utils/replaceMaskText.ts

```diff
@@ -0,0 +1 @@
+export const replaceMaskText = (text: string, maskChar?: string) => text.replace(/\d/g, maskChar || '●');
```

Comment:
```suggestion
export const replaceMaskText = (text: string, maskChar =  '●') => text.replace(/\d/g, maskChar);
```
이렇게 표현하는게 조금 더 자연스러울 것 같네요!